SPECS/azurelinux-repos/azurelinux-repos.spec

```diff
@@ -208,6 +236,9 @@ gpg --batch --yes --delete-keys 2BC94FFF7015A5F28F1537AD0CD9FED33135CE90
 %{_sysconfdir}/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY
 
 %changelog
+* Mon Dec 16 2024 Gary Swalling <gaswal@microsoft.com> - 3.0-4
```

Comment:
You'd still need to update the chroot manifest (see: the last failing PR check).

Reply:
looks like someone added another repo so I rebased, but it also pointed out that I didn't add amd repos to the toolchain_x86_64.txt manifest.